### PR TITLE
update layout to take advantage of semantic ui styles

### PR DESCRIPTION
**Pull request by jwrubel**
Allow me to say that I have no idea what I'm doing here, but I _think_ This shouldn't break the world. Tests pass. Also in a related note, I figured out how to run tests in phoenix/elixir. 

Anyway this just cleans up the menu and styles a little bit.  Okay, a lot. But it should be entirely cosmetic.

### CODE REVIEW

**Comment by jwrubel** [comment]
dev_wizard_-_it_s_a_real_wizard_

This is what a page looks like now.

**Comment by jwrubel** [comment]
Test failure is related to version of elixir? (noidea) Tests were green for me.

File: web/static/js/ui.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$('.message .close')
```
Comment (srobbibaro) [inline web/static/js/ui.js:1]: Do we use this?

**Comment by srobbibaro** [comment]
@jwrubel - rebase with the latest master. I merged a change in which should fix your failing tests.

File: web/static/js/ui.js (hunk 1 above)
Comment (jwrubel) [inline web/static/js/ui.js:1]: I rejiggered the alert boxes to be closeable and use Semantic styles[here](https://github.com/thinkthroughmath/dev_wizard/pull/42/files/7d624abc211841db0284cedb9b9897073a0b1056#diff-4e4c242ad765559995e9844dd68aa05aR44).

**Comment by jwrubel** [comment]
@srobbibaro (highfive) done

**Comment by joelmccracken** [comment]
LGTM! thanks Jim!!!

**Comment by seejee** [comment]
@jwrubel++

**Pull request merged**